packages/turf-meta/index.ts

```diff
@@ -387,7 +455,10 @@ export function propReduce(geojson, callback, initialValue) {
  *   //=featureIndex
  * });
  */
-export function featureEach(geojson, callback) {
+export function featureEach(
```

Comment:
I think this could be :

```
export function featureEach<T extends Geometry>(
  geojson: Feature<T> | FeatureCollection<T>,
  callback: (feature: Feature<T>, featureIndex: number) => boolean | void
): void {
```
Then you won't have to use `as Feature<Point>` etc in other places this is used